examples/download.rs

```diff
@@ -72,18 +76,25 @@ fn main() {
         .unwrap();
 
     let dl = downloader::Download::new(
-        "https://cdimage.debian.org/debian-cd/current/amd64/iso-cd/debian-10.7.0-amd64-netinst.iso",
```

Comment:
`debian-cd/current/` only keeps the latest version, so the URLs go out of date.  They don't seem to archive the netinstaller versions and the ISOs are too large, so I just went looking for a medium-sized test file that would be stable and Hetzner seems to provide one.

Reply:
That is perfectly fine! Thank you!